src/includes/getting-started-primer/react-native.mdx

```diff
@@ -0,0 +1,20 @@
+<Note>
+
+_Sentry's React Native SDK enables automatic reporting of errors and exceptions, and identifies perfomance issues in your application. The React Native SDK includes performance monitoring (our `@sentry/tracing` package) by default._
+
+</Note>
+
+Features:
+
+- Automatic Native Crash [Error Tracking](/product/error-monitoring/) (using both `sentry-cocoa` & `sentry-android`)
+- Offline storage of events
```

Comment:
could be linked to the draining section, noticed RN does not have it.

See https://sentry-docs-git-update-react-native-content.sentry.dev/platforms/android/configuration/draining/

Reply:
it was missing for Apple btw but I just added it https://github.com/getsentry/sentry-docs/pull/3169

Reply:
RN is listed as `notSupported` for the draining info. I'll enable it. Is the Android snippet correct for RN as well? (if so, I'll modify to specify RN, but use that language.)